packages/react-core/src/components/Progress/ProgressContainer.tsx

```diff
@@ -38,7 +38,21 @@ export interface ProgressContainerProps extends Omit<React.HTMLProps<HTMLDivElem
   /** Whether string title should be truncated */
   isTitleTruncated?: boolean;
   /** Position of the tooltip which is displayed if title is truncated */
-  tooltipPosition?: 'auto' | 'top' | 'bottom' | 'left' | 'right';
+  tooltipPosition?:
+    | TooltipPosition
```

Comment:
I've seen this done in a few other places, so this is more of a general question: Why do we include the interface when it has the same values as the strings following it?

Reply:
interface is included for the IDE i guess, while the string values are included because they are necessary for the parser that creates the documentation

Reply:
I do find it redundant to include the interface as well, but I won't consider this a blocking comment.